src/makeInfinityAdder.js

```diff
@@ -1,10 +1,21 @@
 'use strict';
 
-/***
- * @return {function}
- */
-function makeInfinityAdder() {
-  // write code here
+function makeAdder() {
+  let sum = 0;
+
+  return function adder(...numbers) {
+    sum += Number(numbers);
```

Comment:
The use of `Number(numbers)` is incorrect here. `numbers` is an array, and you need to sum its elements. Consider using `numbers.reduce((acc, num) => acc + Number(num), 0)` to correctly sum the values.